ansible/roles/software/ckan/catalog/ckan-app/templates/etc_apache2_sites-enabled_ckan.conf.j2

```diff
@@ -9,22 +9,28 @@ SSLEngine on
     # CSW endpoints provided by pycsw
     # based on request URI
 
+  {% if ckan_uses_gunicorn %}
+    # gunicorn configuration
+    ProxyPass / http://localhost:5000/
+    ProxyPassReverse / http://localhost:5000/
+    ProxyPreserveHost On
+    RequestHeader set X-Forwarded-Proto https
+  {% else %}
```

Comment:
Those CSW lines below should go above Line 12 here. Those proxy to the pycsw service. Not important for now, but I'll add an issue to enable CSW for catalog-next that we can track for later.